creativecloud/scripts/scripts.js

```diff
@@ -163,6 +162,14 @@ const CONFIG = {
  * ------------------------------------------------------------
  */
 
+const observer = new PerformanceObserver((list) => {
+  list.getEntries().forEach((entry) => {
+    console.log(entry);
```

Comment:
@yesil Could you please remove the console log.

Reply:
@aishwaryamathuria good catch, I need to remove the whole `PerformanceObserver`, thanks.